README.md

```diff
@@ -87,7 +87,7 @@ sudo rm /usr/local/bin/eosio-*
 * eosio-objdump
 * eosio-readelf
 
-Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided_installation_or_building_from_scratch)
+Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided-installation-or-building-from-scratch)
 eosio-abidiff
 eosio-ranlib
 eosio-ar
```

Comment:
Make this a bulleted list please. Right now it just is one run big set of words in the final rendering...

Reply:
Fixed.